rush.json

```diff
@@ -1060,13 +1060,13 @@
       "packageName": "@rushstack/module-minifier-plugin",
       "projectFolder": "webpack/module-minifier-plugin",
       "reviewCategory": "libraries",
-      "shouldPublish": false
+      "shouldPublish": true
```

Comment:
Is the publishing issue fixed?

Reply:
If we're going to go with `@rushstack/webpack5-module-minifier-plugin`, we should deprecate+tombstone `@rushstack/module-minifier-plugin` and replace it with `@rushstack/webpack4-module-minifier-plugin`

Reply:
Moving re-enabling this to a separate PR